.ocamlformat

```diff
@@ -0,0 +1,3 @@
+profile = conventional
+version = 0.24.1
+ocaml-version = 4.11
```

Comment:
Is there a strong reason to move the OCaml version this far forward?
https://ocaml.org/p/dockerfile/8.0.0 supports back to 4.02.3

Reply:
We can remove the dependency on rresult, use new functions from the standard library. I think there's a case for moving slowly to new versions. An informal rule is to support the last three OCaml versions. I can go back to 4.10 for now.

Reply:
the CI doesn't currently test before 4.07.